git_machete/docs.py

```diff
@@ -30,6 +30,34 @@
     "version": "Display the version and exit"
 }
 
+github_api_access = '''To allow GitHub API access for private repositories (and also to perform side-effecting actions like opening a PR,
+        even in case of public repositories), a GitHub API token with `repo` scope is required, see https://github.com/settings/tokens.
+        This will be resolved from the first of:
+
+            1. `GITHUB_TOKEN` env var,
+            2. content of the `.github-token` file in the home directory (`~`),
+            3. current auth token from the `gh` GitHub CLI,
+            4. current auth token from the `hub` GitHub CLI.
+    '''
```

Comment:
This string has an embedded indentation, which is probably fine, but maybe it would be better in the long run to dedent this and use `textwrap.indent(…)` at use-site?

Same would apply to `github_config_keys`.

NOTE: In the suggestion here, I haven't applied the other comment from `docs/source/github_config_keys.rst`.

```suggestion
github_api_access = '''To allow GitHub API access for private repositories (and also to perform side-effecting actions like opening a PR,
even in case of public repositories), a GitHub API token with `repo` scope is required, see https://github.com/settings/tokens.
This will be resolved from the first of:

    1. `GITHUB_TOKEN` env var,
    2. content of the `.github-token` file in the home directory (`~`),
    3. current auth token from the `gh` GitHub CLI,
    4. current auth token from the `hub` GitHub CLI.
'''
```